metadata.json

```diff
@@ -49,10 +52,10 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.10.0 < 8.0.0"
+      "version_requirement": ">= 6.0.0 < 9.0.0"
     }
   ],
-  "pdk-version": "2.6.1",
-  "template-url": "pdk-default#2.7.1",
-  "template-ref": "tags/2.7.1-0-g9a16c87"
+  "pdk-version": "3.0.0.pre (1)",
```

Comment:
@MartyEwings sorry to be a pain but is using a pre release PDK a good idea for this?

Reply:
shoot i forgot i had that installed, i dont think this makes much of a difference, the versions of all the linters etc are the same, the actual package i ran shouldn't really mean anything

Reply:
I would wait. It should be within a couple of days.